docs/general-concepts/web-asset-manager.md

```diff
@@ -474,12 +474,12 @@ Will produce:
 
 :::note Note
 
-Named inline asset may be as dependency to another inline asset, hovewer it is not recomended to use an inline asset as dependency to non-inline asset. This will work, but this behavior may changes in future. Prefer to use "position" instead.
+Named inline asset may be as dependency to another inline asset, however it is not recmomended to use an inline asset as dependency to non-inline asset. This will work, but this behavior may changes in future. Prefer to use "position" instead.

```

Comment:
```suggestion
Named inline asset may be as dependency to another inline asset, however it is not recommended to use an inline asset as dependency to non-inline asset. This will work, but this behavior may changes in future. Prefer to use "position" instead.

```